### ISSUE TITLE
Identity generator's `files` option incorrectly references a `list-files` option

### ISSUE THREAD

**Issue by Phrohdoh** [aspnet/Scaffolding#870]
## Steps to reproduce:

```shell
$ dotnet aspnet-codegenerator identity -h
```

The `Generator Options` section has a `--files` option that references a `--list-files` option.

```
Generator Options:
  --dbContext|-dc      : Name of the DbContext to use, or generate (if it does not exist).
  --files|-fi          : List of semicolon separated files to scaffold. Use the --list-files option to see the available options.
  --listFiles|-lf      : Lists the files that can be scaffolded by using the '--files' option.
  --userClass|-u       : Name of the User class to generate.
  --useSqLite|-sqlite  : Flag to specify if DbContext should use SQLite instead of SQL Server.
  --force|-f           : Use this option to overwrite existing files.
  --useDefaultUI|-udui : Use this option to setup identity and to use Default UI.
  --layout|-l          : Specify a custom layout file to use.
  --generateLayout|-gl : Use this option to generate a new _Layout.cshtml
```

This option does not exist.

It is actually `--listFiles`.

### Package version of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All (if applicable):

```xml
    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.3" />
```

### Package version of Microsoft.VisualStudio.Web.CodeGeneration.Design - this may be added to your project by scaffolding:

```xml
    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.3" />
```

### Proposed solution

Fix the following areas:

https://github.com/aspnet/Scaffolding/blob/61114a665ee506804c3a648cb88b8ebc11ae9819/src/VS.Web.CG.Mvc/ParameterDefinitions/identity.json#L14

https://github.com/aspnet/Scaffolding/blob/61114a665ee506804c3a648cb88b8ebc11ae9819/src/VS.Web.CG.Mvc/Identity/IdentityGeneratorCommandLineModel.cs#L19

**Comment by seancpeters** [comment]
Fixed with #874. Many thanks to @t123yh for the fix!

**Comment by seancpeters** [comment]
Also thanks to @Phrohdoh!
